### PR TITLE
Overall update

### PR DESCRIPTION
**Pull request by userMacieG**
ox support
code cleanup + reformatted

### CODE REVIEW

File: .github/contributing.md [hunk 1]
```diff
@@ -63,9 +61,9 @@ Before creating bug reports, please check [this list](#before-submitting-a-bug-r
 #### Before Submitting A Bug Report
 
 * **Check the docs** There's a chance what you see as a bug might just work differently than you expect and if you think it could work better consider a feature enhancement report instead.
-* **Search the [discord](https://discord.gg/qbcore)** to see if anyone else has run into the issue and see if it was solved through user error or code changes. (if the code change isn't pending a PR and you know what you're doing consider submitting one following [Pull Requests](#pull-requests) )
-* **Determine which resource the problem should be reported in**. If the bug is related to the inventory for example report this bug under qb-inventory rather than under qb-core or some other resource.
-* **Perform a [cursory search](https://github.com/search?q=+is%3Aissue+user%3Aqbcore-framework)** to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
```
Comment (ihyajb) [inline .github/contributing.md:68]: This link is still wrong, if you go to the link https://github.com/search?q=+is%3Aissue+user%3Aqbox its random stuff not related to the project.
The correct link is https://github.com/search?q=+is%3Aissue+user%3Aqbox-project

Comment (userMacieG) [reply .github/contributing.md:68]: Good catch

File: client/main.lua [hunk 2]
```diff
@@ -161,36 +177,105 @@ local function busGarage()
...
+                    inRange = false
+                end
+            })
+        end
+    else
+        if VehicleZone then
+            VehicleZone:remove()
+        end
+    end
+end
```
Comment (YvanCywan) [inline client/main.lua:244]:
```suggestion
local function busMain()
    if PlayerData.job.name ~= "bus" then
        if VehicleZone then
            VehicleZone:remove()
        end
        return
    end

    if VehicleZone then return end
    local inRange = false

    VehicleZone = lib.zones.sphere({
        name = "busMain",
        coords = vec3(Config.Location.x, Config.Location.y, Config.Location.z),
        radius = 5,
         debug = false,
        onEnter = function()
            inRange = true

            CreateThread(function()
                repeat
                    Wait(0)

                    local inVeh = whitelistedVehicle()

                    if inVeh then
                        lib.showTextUI(Lang:t('info.bus_stop_work'))

                        if IsControlJustReleased(0, 38) then
                            if not NpcData.Active or NpcData.Active and not NpcData.NpcTaken then
                                if IsPedInAnyVehicle(cache.ped, false) then
                                    BusData.Active = false

                                    DeleteVehicle(cache.vehicle)
                                    RemoveBlip(NpcData.NpcBlip)

                                    lib.hideTextUI()

                                    resetNpcTask()
                                    break
                                end
                            else
                                QBCore.Functions.Notify(Lang:t('error.drop_off_passengers'), 'error')
                            end
                        end
                    else
                        lib.showTextUI(Lang:t('info.busstop_text'))

                        if IsControlJustReleased(0, 38) then
                            busGarage()

                            lib.hideTextUI()
                            break
                        end
                    end
                until not inRange
                end)
            end,
        onExit = function()
            lib.hideTextUI()
            inRange = false
        end
    })
end
```
Organising this function as such will improve readability and save on indentation

File: client/main.lua [hunk 3]
```diff
@@ -161,36 +177,105 @@ local function busGarage()
...
+
             TriggerEvent("vehiclekeys:client:SetOwner", QBCore.Functions.GetPlate(veh))
+
             SetVehicleEngineOn(veh, true, true, false)
         end, data.model, coords, true)
+
         Wait(1000)
+
         TriggerEvent('qb-busjob:client:DoBusNpc')
     end
```
Comment (YvanCywan) [inline client/main.lua:271]: you could actually replace the else on line ```252``` with an end to save on indentation.

File: server/main.lua [hunk 4]
```diff
@@ -1,34 +1,46 @@
...
 end
 
 RegisterNetEvent('qb-busjob:server:NpcPay', function()
     local src = source
     local Player = QBCore.Functions.GetPlayer(src)
-    if not Player then return end
+
+    if not Player then
+        return
+    end
```
Comment (YvanCywan) [inline server/main.lua:26]: Simple guard clauses can stay on one line, imo

**Pull request closed without merging**